packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -78,7 +76,9 @@ const renderTrigger = ({
     onRemove(option);
   };
 
-  const Caret = isOpen ? <CaretUpIcon css={triggerSc.iconStyles} /> : <CaretDownIcon css={triggerSc.iconStyles} />;
+  function renderCaret() {
+    return isOpen ? <triggerSc.UpIcon /> : <triggerSc.DownIcon />;
```

Comment:
I do not like the dependency on `@paprika/list-box/lib/components/Trigger/Trigger.styles` in `ListBoxWithTags` and would like to eliminate it in my follow-up PR that focusses on this component.  

Not sure if I should copy/paste the needed styling, or move it to a helper function in the `@paprika/stylers` package.  Opinions?

Reply:
I'd prefer copy/paste more than coupling stylings between packages. Reusing trigger css just saving some lines of code, but when the changes happening in list box, it might break list box with tags, we'll never know unless we test list box with tags. I just feel there're not many benefits from coupling

Reply:
To be addressed in https://aclgrc.atlassian.net/browse/UXD-868